content/site/about-us.es.md

```diff
@@ -1,4 +1,4 @@
-<p class="text-style-headline-2">Nuestro objetivo es proveer los mejores recursos posibles para aprender cualquier tema relacionado con las ciencias de la computación</p>
+<p><span class="text-style-headline-2">Nuestro objetivo es proveer los mejores recursos posibles para aprender cualquier tema relacionado con las ciencias de la computación</span></p>
```

Comment:
I don't understand this change, could you explain more?

Reply:
Since it's inside a `.post-body`, a `p.text-style-headline-2` would have both the `.post-body p` and `.text-style-headline-2` styles. Because of the `:where(...)` change, the `.post-body p` takes precedence (we want `.text-style-headline-2`).